### ISSUE TITLE
Doubt  test/script_env

### ISSUE THREAD

**Issue by abravalheri** [conda/conda-build#4208]
Hello, I was wondering if `script_env` is supported in the `test` section `meta.yml`, of if the variables specified in `build/script_env` will also be available when running the test command.

**Comment by traversaro** [comment]
Indeed, it would be useful to know how to set environment variables in the test section of conda-build.
